src/controllers/segmentation.py

```diff
@@ -27,8 +40,8 @@ def segment(self, output_file: str):
     def _assign_classes(self):
         for obj in bpy.data.objects:
             if "segmentation_id" in obj:
-                if obj["segmentation_id"] in self.classes:
-                    obj["inst_id"] = self.classes[obj["segmentation_id"]]
+                if obj["segmentation_id"] in self.color_map:
+                    obj["inst_id"] = obj["segmentation_id"]
                 elif obj["segmentation_id"] != 0:
                     print(
                         "WARNING: Unknown segmentation id: "
```

Comment:
Would be better to raise a warning here instead of printing https://docs.python.org/3/library/warnings.html#warnings.warn